README.md

```diff
@@ -89,11 +96,21 @@ FAQ
 
 3. What can I run on Origin?
 
-    Origin is designed to run any existing Docker images.  In addition you can define builds that will produce new Docker images from a Dockerfile.  However the real magic of Origin can be seen when using [Source-To-Image](https://github.com/openshift/source-to-image) builds which allow you to simply supply an application source repository which will be combined with an existing Source-To-Image enabled Docker image to produce a new runnable image that runs your application.  We are continuing to grow the ecosystem of Source-To-Image enabled images and documenting them [here](http://docs.openshift.org/latest/using_images/s2i_images/overview.html). Our available images are:
+    Origin is designed to run any existing Docker images.  Additionally, you can define builds
+    that will produce new Docker images using a `Dockerfile`.
+
+    However, the real magic of Origin is [Source-to-Image (S2I)](https://github.com/openshift/source-to-image)
```

Comment:
I think there's more too Origin than S2I.  *grin*.  Maybe "the real magic of Origin can be illustrated by" or similar?